demo/site/package.json

```diff
@@ -5,10 +5,12 @@
     "description": "Comet Site Demo Application",
     "type": "commonjs",
     "scripts": {
-        "build": "run-s intl:compile && run-p gql:types generate-block-types build-server && next build",
+        "build": "run-s intl:compile && run-p gql:types generate-block-types css:types build-server && next build",
         "build-server": "tsc --project tsconfig.server.json",
         "check-node-version": "check-node-version --node $(cat ../../.nvmrc)",
-        "dev": "$npm_execpath check-node-version && run-s intl:compile && run-p gql:types generate-block-types build-server && dotenv -e .env.secrets -e .env.local -e .env -e .env.site-configs -- node --inspect=localhost:9230 --max-old-space-size=512 dist/server.js",
+        "css:types": "typed-scss-modules src",
+        "css:types:watch": "typed-scss-modules src --watch",
```

Comment:
We should start this when starting the site (either in the dev script or dev-pm config), otherwise new classes won't be generated.

Reply:
yes, that was missing